meinberlin/apps/plans/models.py

```diff
@@ -196,9 +196,7 @@ def administrative_district(self):
 
     @property
     def topic_names(self):
-        if hasattr(settings, "A4_PROJECT_TOPICS"):
-            return [topic.name for topic in self.topics.all()]
-        return []
+        return [topic.__str__() for topic in self.topics.all()]
```

Comment:
@goapunk I will change these to str(topic) so we don't call private functions. and the __str__ one is specifically for the output of the instance when called with print() and str().

Reply:
@m4ra yes, good idea